public/app/core/utils/errors.ts

```diff
@@ -60,3 +60,10 @@ export function getMessageIdFromError(err: unknown): string | undefined {
 
   return undefined;
 }
+
+export function getRequestConfigFromError(err: FetchError): string {
+  const method = err.config?.method ?? 'GET';
```

Comment:
are we defaulting to `GET` because we know that is the default value in case of missing this method field?

Reply:
Yep. `GET` is the default method when it's left blank :)